windows/vtpm_cold_add_remove/bitlocker_enable_test.yml

```diff
@@ -0,0 +1,113 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Check C: drive used space is encrypted, protection is on,
+# and encryption key protector is TPM after enabling BitLocker
+#
+# From Windows 11 v24H2, hardware requirements for Automatic Device Encryption are:
+# TPM 2.0 and UEFI secure boot,
+# please refer to this doc: https://learn.microsoft.com/en-us/windows-hardware/design/device-experiences/oem-bitlocker
+- name: "Get VM firmware info for Windows client"
+  include_tasks: ../../common/vm_get_boot_info.yml
+  when: guest_os_product_type == 'client'
+
+- name: "Set fact of whether BitLocker Drive Encryption is enabled automatically"
+  ansible.builtin.set_fact:
+    win_bde_auto_enable: "{{ guest_os_product_type == 'client' and guest_os_ansible_distribution_ver is version('10.0.26100.0', '>=') and vm_secureboot_enabled }}"
+
+# BitLocker is not installed by default in Windows Server
+- name: "Check and install BitLocker feature in Windows Server"
```

Comment:
What about the Windows Client earlier than Win11 24H2 (26100)?  Do they need to install the bitlocker feature?

Reply:
No, the feature installed in Windows client. Although the edition matters, we can also check the edition here.